presto-product-tests/conf/docker/common/standard.yml

```diff
@@ -15,6 +15,18 @@ services:
     volumes:
       - ../../../../:/docker/volumes/presto
 
+  presto-master-ldap:
+    image: 'teradatalabs/centos6-java8-oracle-ldap'
+    hostname: presto-master
+    env_file:
+      - ../../../target/classes/presto.env
+    volumes:
+      - /etc/ldap/rootca
```

Comment:
Sorry, didn't get that, which catalog. If you are talking about the volumes, that path exists in image.